src/Sprites/spriteMap.ts

```diff
@@ -0,0 +1,627 @@
+import { Engine } from "../Engines/engine";
+import { IDisposable, Scene } from "../scene";
+import { Nullable } from "../types";
+import { Vector2, Vector3 } from "../Maths/math.vector";
+//import { Color3 } from "../Maths/math.color";
+//import { Camera } from "../Cameras/camera";
+import { Texture } from "../Materials/Textures/texture";
+import { RawTexture } from "../Materials/Textures/rawTexture";
+import { ShaderMaterial } from "../Materials/shaderMaterial";
+import { Mesh } from "../Meshes/mesh";
+import { Observer } from "../Misc/observable";
+import { ExecuteCodeAction } from "../Actions/directActions";
+import { ActionManager } from "../Actions/actionManager";
+import { PickingInfo } from "../Collisions/pickingInfo";
+
+import "../Meshes/Builders/planeBuilder";
+
+/**
+ * Defines the minimum interface to fulfill in order to be a sprite map.
+ */
+export interface ISpriteMap extends IDisposable {
+

```

Comment:
seems empty :)

Reply:
You also need to include the ts file for your shaders or they will be tree shacked by webpack

Reply:
I do not see the import for shaders?